mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/dependencies/StrategyDependencyResolver.java

```diff
@@ -93,4 +116,8 @@ private Properties loadFromFile() {
                     return dependency;
                 }));
     }
+
+    private String filterPrefix(String key) {
+        return key.substring(key.lastIndexOf(SMART_TESTING_STRATEGY_PREFIX) + SMART_TESTING_STRATEGY_PREFIX.length());
```

Comment:
why "last index of" - I think that this is for multiple occurrences which is not this case.
In addition, wouldn't be enough to just call `key.substring(SMART_TESTING_STRATEGY_PREFIX.length())`? I haven't tried it locally, but I believe that something like this should work...